core/config.go

```diff
@@ -45,23 +45,40 @@ func NewChainConfig() *ChainConfig {
 	}
 }
 
-// IsHomestead returns whether num is either equal to the homestead block or greater.
+// IsHomestead returns whether num is greater than or equal to the homestead block, but less than gotham.
 func (c *ChainConfig) IsHomestead(num *big.Int) bool {
 	if c.Fork("Homestead").Block == nil || num == nil {
 		return false
 	}
 	return num.Cmp(c.Fork("Homestead").Block) >= 0
 }
 
+
+// IsGotham returns whether num is greater than or equal to the gotham block, but less than explosion.
+func (c *ChainConfig) IsGotham(num *big.Int) bool {
+    if c.Fork("Gotham").Block == nil || num == nil {
+        return false
+    }
+    return num.Cmp(c.Fork("Gotham").Block) >= 0 && num.Cmp(c.Fork("Explosion").Block) < 0
+}
+
+// IsExplosion returns whether num is either equal to the explosion block or greater.
+func (c *ChainConfig) IsExplosion(num *big.Int) bool {
+    if c.Fork("Explosion").Block == nil || num == nil {
+        return false
+    }
+    return num.Cmp(c.Fork("Explosion").Block) >= 0
+}
+
 func (c *ChainConfig) Fork(name string) *Fork {
-	for i := range c.Forks {
```

Comment:
Don't think it's good idea to change to spaces here

I'm spaces-guy too, but 

1. whole project is written with tabs 
2. tabs are standard formatting for Go 
3. don't think it's related to Bomb Delay in any way